kythe/go/extractors/gcp/README.md

```diff
@@ -60,3 +61,28 @@ gcr.io/kythe-public/build-preprocessor is just
 [kythe/go/extractors/config/preprocessor](https://github.com/google/kythe/blob/master/kythe/go/extractors/config/preprocessor/preprocessor.go),
 which we use to preprocess the `pom.xml` build configuration to be able to
 specify all of the above custom javac extraction logic.
+
+## Troubleshooting
+
+### Generic failure to use gcloud
+
+Make sure you've followed the setup setps above in [Cloud Build](#cloud-build),
+especially `gcloud auth login`.
+
+### Step #N: fatal: could not read Username for 'https://github.com': No such device or address
+
+This, confusingly, could be two completely separate errors.  First, and simpler
+to check, you could have just spelled the repo incorrectly.  If you have a
+typo in the repo name, instead of telling you "repo doesn't exist" or something,
+the failure message is the above error about "could not read Username".
+
+If you have verified that the repo name is spelled correctly, then you may be
+trying to access a private git repo.  This is still possible, but you need to
```

Comment:
Replace the "This" in "This is still possible" since it is a little ambiguous.

Reply:
Yea I suppose that's true.